source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst

```diff
@@ -279,13 +279,35 @@ Auto partition will configure the media to meet the minimum requirements.
 
 #. You may skip to the :ref:`telemetry-tui` section below.
 
+Disk encryption
+===============
+
+For greater security, disk encryption is supported by use of LUKS for all
+partitions **except boot** on |CL|. Encryption is optional. To encrypt a
+partition, see the example below to encrypt :ref:`encrypt-root`.
+
+Encryption Passphrase
+---------------------
+
+|CL| supports use of a single passphrase for all encrypted partitions.
+Therefore, if you turn on encryption for one partition, the same passphrase
+is used for all partitions.
+
+.. note::
+
+   For example, suppose you encrypt the `root` partition first and the `swap`
+   partition second. Upon selecting encryption for the `swap` partition, the
+   :guilabel:`Encryption Passphrase` dialogue appears showing the default
+   Passphrase assigned to the `root`. Select :guilabel:`Confirm` and press
+   :kbd:`Enter` to use the same Passphrase.
+
```

Comment:
**Additional or different keys may be configured post-installation using `cryptsetup` tool.**